src/exec/src/lib.rs

```diff
@@ -5,7 +5,9 @@ extern crate protobuf;
 #[macro_use]
 extern crate log;
 
+#[rustfmt::skip]
```

Comment:
Could you shed some light on why skip format on the two modules?

Reply:
Done.